pkg/metrics/metrics.jsx

```diff
@@ -1622,7 +1640,23 @@ class MetricsHistory extends React.Component {
 
         const isBeibootBridge = cmdline?.includes("ic# cockpit-bridge");
 
-        this.setState({ packagekitExists, isBeibootBridge });
+        let boots = [];
+        try {
+            // Only 14 days of metrics are shown
+            const output = await cockpit.spawn(["journalctl", "--list-boots", "--since", "-15d", "--output", "json"]);
+            const list_boots = JSON.parse(output);
+            boots = list_boots.map(boot => {
+                return {
+                    started: new Date(boot?.first_entry / 1000),
+                    ended: new Date(boot?.last_entry / 1000),
+                    current_boot: boot?.index === 0,
+                };
+            });
+        } catch (exc) {
+            console.warn("journalctl --list-boots failed", exc);
+        }
+
+        this.setState({ packagekitExists, isBeibootBridge, boots });
```

Comment:
Please move this into the try block (no need to set `boots` if it failed to scan), and only set `boots` here, see above.